src/Runner/FileLintingIterator.php

```diff
@@ -21,6 +21,8 @@
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  *
  * @internal
+ *
+ * @extends \IteratorIterator<mixed, \SplFileInfo, \Traversable<\SplFileInfo>>
```

Comment:
what is the first type here, for? maybe we can avoid `mixed` here?

Reply:
It's mandatory. I used `mixed` because we don't actually care about the key type, but I might change it if you want.